contrib/opencensus-ext-django/CHANGELOG.md

```diff
@@ -5,8 +5,9 @@
 ## 0.7.0
 Released 2019-07-31
 
-- Updated span attributes to include some missing attributes listed [here](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md#attributes)
-([#735](https://github.com/census-instrumentation/opencensus-python/pull/735))
+- Updated span attributes to include some missing attributes listed
```

Comment:
Need to update version files for these extensions: `opencensus-ext-django`, `opencensus-ext-flask`, `opencensus-ext-jaeger`, `opencensus-ext-ocagent`, `opencensus-ext-pymongo`.